modules/user_accounts/jsx/userAccountsIndex.js

```diff
@@ -87,10 +96,18 @@ class UserAccountsIndex extends Component {
     return result;
   }
 
+  /**
+   * Changes url to add User.
```

Comment:
```suggestion
   * Changes url to edit User.
```

Reply:
It's the same url for both adding and editing. So perhaps `add/edit`?